### PR TITLE
Spencer.upgrade okhttp

### PR DESCRIPTION
**Pull request by spencergibb**
For a matching upgrade in https://github.com/spring-cloud/spring-cloud-netflix/issues/1752

### CODE REVIEW

File: ribbon/src/test/java/feign/ribbon/RibbonClientTest.java [hunk 1]
```diff
@@ -127,7 +124,7 @@ public void ioExceptionRetry() throws IOException, InterruptedException {
   @Test
   public void ioExceptionFailsAfterTooManyFailures() throws IOException, InterruptedException {
     server1.enqueue(new MockResponse().setSocketPolicy(SocketPolicy.DISCONNECT_AT_START));
-    server1.enqueue(new MockResponse().setBody("success!"));
+    //server1.enqueue(new MockResponse().setBody("success!"));
```
Comment (adriancole) [inline ribbon/src/test/java/feign/ribbon/RibbonClientTest.java]: just delete the line?

**Comment by adriancole** [comment]
needs rebase and nit.. otherwise gtg

**Comment by spencergibb** [comment]
@adriancole I'm really confused at how this was merged, but it is and that commented out test was removed. I'd like to do a release this week to this can be in the Dalston Spring Cloud release.

**Comment by adriancole** [comment]
heh ok :)

**Comment by spencergibb** [comment]
It was wierd, I did a rebase, then force push and there were 0 commits and it was merged. Anyway, this will line up the version of okhttp among a few projects.

**Pull request closed without merging**